src/main/java/com/mraof/minestuck/entity/ai/attack/AnimatedAttackWhenInRangeGoal.java

```diff
@@ -75,7 +78,7 @@ public void tick()
 	{
 		super.tick();
 		
-		if(time == phasedAnimation.getContactStartTime())
+		if(time == Math.floor(phasedAnimation.getContactStartTime(phasedAnimation.getAnimationSpeed(entity))))
```

Comment:
```suggestion
		if(time == phasedAnimation.getContactStartTime(phasedAnimation.getAnimationSpeed(entity)))
```
No need to floor an int.